pkg/dsl/definition/template.go

```diff
@@ -119,6 +121,14 @@ func (td *traitDef) Complete(ctx process.Context) error {
 			return errors.WithMessagef(err, "traitDef %s build", td.name)
 		}
 
+		processing := inst.Lookup("processing")
+		var err error
+		if processing.Exists() {
+			if inst, err = task.Process(inst); err != nil {
```

Comment:
why not just pass the "http part" instead?

Reply:
I should make use of "inst" in Process method